skycultures/modern/index.json

```diff
@@ -1675,6 +1675,53 @@
       "common_name": {"english": "Sails", "native": "Vela"}
     }
   ],
+  "zodiac": { "name": { "native": "Signa Zodiacala", "english": "Zodiacal signs"},
+              "partitions": [ 12, 30],
+              "extent": 8,
+	      "names": [ { "symbol": "\u2648", "native": "Aries", "english": "Ram"},
+			 { "symbol": "\u2649", "native": "Taurus", "english": "Bull"},
+			 { "symbol": "\u264A", "native": "Gemini", "english": "Twins"},
+			 { "symbol": "\u264B", "native": "Cancer", "english": "Crab"},
+			 { "symbol": "\u264C", "native": "Leo", "english": "Lion"},
+			 { "symbol": "\u264D", "native": "Virgo", "english": "Virgin"},
+			 { "symbol": "\u264E", "native": "Libra", "english": "Scales"},
+			 { "symbol": "\u264F", "native": "Scorpio", "english": "Scorpion"},
```

Comment:
the name is "Scorpius" (Latin, not Italian!)

Reply:
I wanted to look a bit medieval here, whatever is commonly found on old prints. Can be reversed of course.